### DISCUSSION TITLE
Update preview content

### DISCUSSION THREAD

**Discussion by saeldion** [atlassian/pragmatic-drag-and-drop#121] [Q&A]
I'm thinking of migrating our current drag & drop solution to pragmatic since it seems well maintained and I like to approach the lib is taking.

I however have one requirement which I'm not certain pragmatic supports or if I'm just missing something in the documentation or my experimentation.

I need the ability to update the preview element as it is being dragged. The draggable should be dragged across a timeline (across hours and even days) and I want the preview to update its label to reflect the change in time and/or date. Is this possible?  
It looks like the preview is generated once and is never touchable again but I might be missing something.

I appreciate any help or guidance any of you can provide 🙂

**Comment by ru-van-urk** [comment]
@saeldion Did you explore this further? I am curious if you found a solution.

**Comment by saeldion** [comment]
Unfortunately no, I didn’t find a solution. I did some digging and experimenting but I couldn’t  find a way to achieve what I wanted. 
So in the end I wrote my own drag & drop implementation using PointerEvents and so far it is doing what I need it to do. It’s kinda hacked together and will probably bite me in the ass one day. But for now it gets the job done 😅

**Reply by ru-van-urk** [reply]
Thanks for your response. I'll keep hacking at it with this library.

**Comment by hungify** [comment]
Hey @ru-van-urk @saeldion!
I don't understand your expectations well.
However, this may relate to your situation.
Trying to apply the technique in the following guide: [renders a 1x1 element as the native drag preview](https://atlassian.design/components/pragmatic-drag-and-drop/core-package/adapters/element/drag-previews#non-native-custom-drag-previews).

You can check it out in my [repo](https://github.com/hungify/pragmatic-custom-preview) and [example](https://pragmatic-custom-preview.netlify.app/).

**Reply by saeldion** [reply]
Good to know there is a workaround!
